music-playlist-app/index.js

```diff
@@ -83,8 +86,19 @@ function displaySong(song = "", artist = "") {
 }
 
 function playPauseSong() {
-  isPlaying ? song.pause() : song.play();
-
+  if (isPlaying) {
+    song.pause();
+    playPauseBtn.setAttribute(
+      "aria-label",
+      `Paused ${playlist[currentSong].title}`
```

Comment:
```suggestion
      `Play ${playlist[currentSong].title}`
```
I know this seems a little counter intuitive, but the button needs to describe what it will do, not the state of the player. So when the song is paused then the button will play the song and thus the accessible name should be "Play [song title]".